python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -313,6 +313,10 @@ def find_actual(target_name, fetch_list):
                     idx = find_actual(sub_out_name, fetch_list)
                     actual = outs[idx]
                     actual_t = np.array(actual)
+                    # paddle float16 is exposed to python as uint16 type
+                    # reinterpret the memory as numpy.float16
```

Comment:
Why not just expose to python as float16 directly?

Reply:
Done.